packages/shared/src/utils/config/config.ts

```diff
@@ -52,31 +52,29 @@ export const getAppId = () => {
     const current_domain = getCurrentProductionDomain() || '';
     window.localStorage.removeItem('config.platform'); // Remove config stored in localstorage if there's any.
     const platform = window.sessionStorage.getItem('config.platform');
-    const { is_pathname_bot, is_config_route_bot } = isBot();
+    const domain_app_id = domain_app_ids[current_domain as keyof typeof domain_app_ids];
+    const _app_id = isBot() ? 19112 : domain_app_id;
+
     // Added platform at the top since this should take precedence over the config_app_id
     if (platform && platform_app_ids[platform as keyof typeof platform_app_ids]) {
         app_id = platform_app_ids[platform as keyof typeof platform_app_ids];
-    } else if (config_app_id) {
+    }
+    if (config_app_id) {
         app_id = config_app_id;
-    } else if (user_app_id.length) {
+    }
+    if (user_app_id.length) {
         window.localStorage.setItem('config.default_app_id', user_app_id);
         app_id = user_app_id;
-    } else if (isStaging()) {
+    }
+    if (isStaging()) {
         window.localStorage.removeItem('config.default_app_id');
-        app_id =
-            (is_pathname_bot || is_config_route_bot
-                ? 19112
-                : domain_app_ids[current_domain as keyof typeof domain_app_ids]) || 16303; // it's being used in endpoint chrome extension - please do not remove
+        app_id = _app_id || 16303; // it's being used in endpoint chrome extension - please do not remove
```

Comment:
@rupato-deriv I think it would be better to use nullish coalescing operator `??` in here instead of the `or` operator